jsk_fetch_robot/jsk_fetch_startup/euslisp/navigation-utils.l

```diff
@@ -9,6 +9,21 @@
 (defparameter *spots* nil)
 (defparameter *dock-spot* "/eng2/7f/room73B2-fetch-dock-front")
 (defparameter *is-charging* nil)
+(defparameter *tfl* (instance ros::transform-listener :init))
```

Comment:
please add your `*ri*` policy here....

Reply:
I have no policy about `*ri*`.
This PR does not have  any policy about `*ri*`.
What I want to do is to do `:clear-costmap` before `:move-to`.

Reply:
https://github.com/jsk-ros-pkg/jsk_robot/pull/1160#issuecomment-546595519 changed the code based on policy, please add that to source code, also update https://github.com/jsk-ros-pkg/jsk_robot/pull/1160#issue-332598336. 
we would like to why this commit makes auto dock work correctly, too.

Reply:
I mean https://github.com/jsk-ros-pkg/jsk_robot/pull/1160#issuecomment-546595519 is totally misunderstanding.
And I update this PR after the comment.